src/components/Profile/components/PersonalInfoList/index.jsx

```diff
@@ -355,6 +380,135 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
                     {showMailCombo ? mailCombo : '****'}
                   </Typography>
                 </Grid>
+                <Grid item md={3}>
+                  <div className={styles.header_tooltip_container}>
+                    <CustomTooltip
+                      disableFocusListener
+                      disableTouchListener
+                      title={
+                        // eslint-disable-next-line no-multi-str
+                        <Grid container>
+                          <Grid item>
+                            <Typography sx={{ fontSize: '1.0rem' }}>
+                              {
+                                'Salsbury Mailbox (Combinations that have three numbers ex: 21 32 18)'
+                              }
+                            </Typography>
+                            <img src={SLock} alt="SLock" />
+                          </Grid>
+                          <Grid item>
+                            <Typography sx={{ fontSize: '0.8rem' }}>
+                              {
+                                '1) To openturn LEFT at least four turns stopping at the first number of\
+                               the combination.'
+                              }
+                            </Typography>
+                          </Grid>
+                          <Grid item>
+                            <Typography sx={{ fontSize: '0.8rem' }}>
+                              {
+                                '2) Turn RIGHT passing the first number of the combination once and stop at\
+                                 the second number of the combination.'
+                              }
+                            </Typography>
+                          </Grid>
+                          <Grid item>
+                            <Typography sx={{ fontSize: '0.8rem' }}>
```

Comment:
These `fontSize` styles make for a lot of repeated code. Is there a better way to do this?

Reply:
I was only focused on getting things to work yesterday, I will try to see if there is a more efficient way to do this. Thank you for the feedback.